data/aliases.ts

```diff
@@ -5,7 +5,7 @@ export const Aliases: {[alias: string]: string} = {
 	mono: "[Gen 8] Monotype",
 	ag: "[Gen 8] Anything Goes",
 	bss: "[Gen 8] Battle Stadium Singles",
-	vgc: "[Gen 8] VGC 2020",
+	vgc: "[Gen 8] VGC 2021",
```

Comment:
this wont do anything since theres no format named vgc 2021, you can alias this to series 9 if youd like